context/app/static/js/components/Search/TilesSortDropdown/TilesSortDropdown.jsx

```diff
@@ -1,78 +1,48 @@
-import React, { useRef, useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import Paper from '@material-ui/core/Paper';
-import Popper from '@material-ui/core/Popper';
-import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDownRounded';
-import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUpRounded';
-import MenuList from '@material-ui/core/MenuList';
-import ClickAwayListener from '@material-ui/core/ClickAwayListener';
 
+import DropdownListbox from 'js/shared-styles/dropdowns/DropdownListbox';
 import useSearchViewStore from 'js/stores/useSearchViewStore';
-import { SelectionButton, StyledDropdownSelectItem } from './style';
+import { StyledButton, StyledDropdownListboxOption } from './style';
 import { getSortPairs } from '../utils';
 
 const searchViewStoreSelector = (state) => state.searchView;
 
-function getSelectedItemLabel(items, selectedItems) {
+function getSelectedItemIndex(items, selectedItems) {
```

Comment:
So is the question what does the function do? If that's the case, I can add a comment. To clarify here, Searchkit provides lists of items and selectedItems as props, `getSelectedItemIndex` returns the index of the selected item in the list of items. Or is the question something else?

Reply:
Renaming the parameter may be enough: On the first review, it was hard to understand what was going on, because it wasn't clear that the inputs were different.